ffi/node/Makefile

```diff
@@ -1,6 +1,13 @@
 ZKGROUP_RUST_DIR=../../rust
 ZKGROUP_TARGET_DIR=../../target
 
+NODE_ARCH := $(shell node -p "process.arch" || x64)
```

Comment:
`|| echo x64`?

Reply:
Good catch! Just tested locally and indeed needs the `echo` 👍🏼